src/sardana/pool/poolcontrollers/DummyMotorController.py

```diff
@@ -165,12 +169,15 @@ def getAccelerationTime(self):
     def setDecelerationTime(self,dt):
         """Sets the time to go from maximum velocity to minimum velocity in seconds"""
         dt = float(dt)
-        if dt <= 0:
-            raise "Deceleration time must be > 0"
+        if dt < 0:
+            dt = 0
 
         self.decel_time = dt
-        self.decel = (self.min_vel - self.max_vel) / dt
-
+        if dt != 0:
+            self.decel = (self.min_vel - self.max_vel) / dt
+        else:
+            self.decel = float ('INF')
```

Comment:
I think that instead of self.decel=float('INF') in this case it should be self.decel=float('-INF').
It has not any major implication in the code, but the sign should be negative because it is a deceleration.